de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -447,6 +462,81 @@ public void onSuccess(Splittable history) {
 
     }
 
+    @Override
+    public void getVICELogs(String id,
+                            String analysisName) {
+        viceLogsAnalysisId = id;
+        viceLogsAnalysisName = analysisName;
+        getLogs(id, analysisName);
+
+    }
+
+    protected void getLogs(String id,
+                           String analysisName) {
+        analysisService.getVICELogs(id, viceLogsSinceTime, new AnalysisCallback<String>() {
+
+            @Override
+            public void onFailure(Integer statusCode,
+                                  Throwable exception) {
+                announcer.schedule(new ErrorAnnouncementConfig(exception.getMessage()));
```

Comment:
During `onSuccess`, we call `viceLogsView.load` or `viceLogsView.update` and that sets the `loading` prop to false to get rid of the loading mask.  In the case where we hit, `onFailure`, the loading mask will still be there.